assets/scss/base.scss

```
@@ -0,0 +1,100 @@
+}
+
+/* ── Code blocks (from code.css) ── */
+
+code {
+  padding: 2px 5px;
+  font-family: monospace;
+}
+
+pre {
+  @apply rounded p-4 overflow-x-auto text-sm;
+  max-width: 100%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)